ocs/defaults.py

```diff
@@ -71,6 +70,25 @@
     'rook_image': ROOK_IMAGE,
 }
 
+DEPLOYMENT = {
```

Comment:
@zmc  Didn't get why those section were moved here? They are not used from the defaults in any places from the code but from ocsci/config.py.

My intention was to remove a lot of variable from here and use them from ocsci/config.py directly which are loaded from the default_config.yaml as I mentioned in the comment in this file. But as we still had some support for old runner I didn't do this cleanup yet and wanted to do it in the PR once I am back from PTO.

This is comment you can see in this file.
```python
# Those variables below are duplicate at the moment from default_config.yaml
# and once we drop support for old runner we will remove those variables from
# here and will have them defined only on one place.

# Be aware that variables defined above and below are not used anywhere in the
# config files and their sections when we rendering config!
```